_posts/0000-01-02-satishwarkedas.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Wecome to our second slide!"
+---
+This is an interesting way to learn Github
```

Comment:
Well not only is this an interesting way to learn, but also an intriguing way to collaborate with myself!